.github/ISSUE_TEMPLATE/accessibility.md

```diff
@@ -2,7 +2,7 @@
 name: Accessibility
 about: Report an accessibility issue
 title: ''
-labels: accessibility general
+labels: 'accessibility-general'
```

Comment:
@maxkadel can you change this to 'accessibility-awaiting-triage' it's the label that we should use for new accessibility tickets if we have not triaged or identify yet the accessibility level issue

Reply:
'accessibility-general' is the label we use when we have triaged the ticket and we have not yet identified the level of the accessibility issue or there is no one to identify .

Reply:
Thanks, I was wondering about that. Changed.